package.json

```diff
@@ -32,7 +32,9 @@
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-burger-menu": "^2.6.13",
-    "react-helmet": "^6.1.0"
+    "react-helmet": "^6.1.0",
+    "react-slick": "^0.27.12",
+    "slick-carousel": "^1.8.1"
```

Comment:
Sorry, could you find a different carousel library? It looks like slick-carousel depends on jQuery, which I don't want to pull in as a dependency into this project, since it's older technology and most of its benefits are no longer relevant due to browsers having evolved a lot over the last decade and incorporating many of jQuery's features natively.

Reply:
👍 Changed to using nuka-carousel instead.